.github/workflows/ett.yml

```diff
@@ -1,7 +1,7 @@
 name: Energy Track and Trace
 
 env:
-  app-name: ett-app
+  app-name: app-ett
```

Comment:
issue(blocking): when this is changed, it affects the Docker image name in Line 50 which is used for deployment, effectively breaking ETT's infrastructure instantly

Reply:
We should properly talk about this then, I did change the path of the dist in the docker file but if there's more depending on the app-name which is not part of the repo I need some help.